modules/helfi_eu_cookie_compliance/config/optional/language/sv/eu_cookie_compliance.cookie_category.statistics.yml

```diff
@@ -1,4 +1,4 @@
bsp;</td></tr><tr><td>_pk_id.*</td><td>hel.fi</td><td>Matomo-statistiksystemets kaka samlar information om hur webbplatsen används.</td><td>393 dagar</td><td>&nbsp;</td></tr><tr><td>_pk_ses.141.89f6</td><td>hel.fi</td><td>-</td><td>1 timme</td><td>&nbsp;</td></tr><tr><td>_pk_id.*</td><td>kartta.hel.fi</td><td>Matomo-statistiksystemets kaka samlar information om hur webbplatsen används.</td><td>393 dagar</td><td>&nbsp;</td></tr><tr><td>_pk_ses.*</td><td>kartta.hel.fi</td><td>Matomo-statistiksystemets kaka samlar information om hur webbplatsen används.</td><td>1 timme</td><td>&nbsp;</td></tr><tr><td>_pk_id.*</td><td>palvelukartta.hel.fi</td><td>Matomo-statistiksystemets kaka samlar information om hur webbplatsen används.</td><td>393 dagar</td><td>&nbsp;</td></tr><tr><td>_pk_ses.*</td><td>palvelukartta.hel.fi</td><td>Matomo-statistiksystemets kaka samlar information om hur webbplatsen används.</td><td>1 timme</td><td>&nbsp;</td></tr><tr><td>rnsbid</td><td><span>reactandshare.com</span></td><td><p><span>En post relaterad till driften av reaktionsknappen&nbsp;</span></p><p><span>Askem</span></p><p><span>.</span></p></td><td>-</td><td><span>Lokalt minne</span></td></tr><tr><td>rnsbid_ts</td><td><span>reactandshare.com</span></td><td><p><span>En post relaterad till driften av reaktionsknappen&nbsp;</span></p><p><span>Askem</span></p><p><span>.</span></p></td><td>-</td><td><span>Lokalt minne</span></td></tr><tr><td>rns_reaction_*</td><td><span>reactandshare.com</span></td><td><p><span>En post relaterad till driften av reaktionsknappen&nbsp;</span></p><p><span>Askem</span></p><p><span>.</span></p></td><td>-</td><td><span>Lokalt minne</span></td></tr><tr><td>YSC</td><td>youtube.com</td><td>YouTubes kaka gör det möjligt att göra videor till en del av innehållet på webbplatsen.</td><td>session</td><td>Tredje part</td></tr><tr><td><span>null_active_survey</span></td><td>Norstat</td><td><span dir="ltr" role="presentation">Id för undersökningen som är live.</span></td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td><span>null_page_view_count</span></td><td>Norstat</td><td><span dir="ltr" role="presentation">Hur många Helsingfors stads sidor har du tittat på innan du såg undersökningen.</span></td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td><span>null_active_scenario</span></td><td>Norstat</td><td><span dir="ltr" role="presentation">Vilken version av undersökningsmodulen är live.</span></td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td><span>wsLanguage_p933511862344</span></td><td>Norstat</td><td><span dir="ltr" role="presentation">Språkidentifierare som används för att välja vilket språk i undersökningen du ser.</span></td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td><span>p933511862344_status</span></td><td>Norstat</td><td>Har du redan fyllt i undersökningen.</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>p933511862344_complete_date</td><td>Norstat</td><td>Datum och tid då enkäten lämnades in.</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></figure><p>&nbsp;</p>'
```

Comment:
The "local storage" / "Lokalt minne" type is missing from the swedish descriptions.